packages/ibm-products/src/components/ProductiveCard/ProductiveCard.tsx

```diff
@@ -51,17 +51,10 @@ export interface ProductiveCardProps extends PropsWithChildren {
    * Determines if the action icons are on the top or bottom of the card
    */
   actionsPlacement?: PlacementType;
-  /**
-   * Content that shows in the body of the card
-   */
-  // children: PropTypes.node,
   /**
    * Optional user provided class
    */
   className?: string;
-
-  children: ReactNode;
-
```

Comment:
Isn't this just removing it from the types? Wouldn't it be better to do this?
```
children?: ReactNode;
```

Reply:
Extending propswithchildren implicitly does that, no? I guess I never considered the nuances of one versus the other. If im wrong please let me know!

Reply:
based on this [article](https://blog.logrocket.com/react-children-prop-typescript/) i think either is fine to use. the [definition of PropWithChildren](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d076add9f29db350a19bd94c37b197729cc02f87/types/react/index.d.ts#L822) is literally that line